Makefile

```diff
@@ -13,8 +13,8 @@
 
 
 IMAGE_NAME := instrumentisto/gitlab-builder
-VERSION ?= 0.4.2-docker18.05.0-compose1.21.2-kubectl1.10.2-helm2.9.1
-TAGS ?= 0.4.2-docker18.05.0-compose1.21.2-kubectl1.10.2-helm2.9.1,0.4.2,0.4,latest
```

Comment:
@alnikyur why did you change Docker CLI version here if MR is related only to Kubernetes CLI?

Reply:
@tyranron See on your [commit](https://github.com/instrumentisto/gitlab-builder-docker-image/commit/7a4a4d17c9fead00d4c7ba00080838d5a9144383) and think about update, but now understood that is my mistake. I`ll fix it now.